tests/performance/src/main/java/org/mule/runtime/core/processor/strategy/ProcessingStrategyBenchmark.java

```diff
@@ -0,0 +1,202 @@
+  private Sink workQueueAllSink;
+
+  @Setup(Level.Trial)
+  public void setUp() throws MuleException {
+    muleContext = createMuleContextWithServices();
+
+    directPs = new DirectProcessingStrategyFactory().create(muleContext, "direct_mb");
+    startIfNeeded(directPs);
+    emitterPs = new TransactionAwareProactorStreamEmitterProcessingStrategyFactory().create(muleContext, "emitter_mb");
+    startIfNeeded(emitterPs);
+    workQueuePs = new TransactionAwareProactorStreamWorkQueueProcessingStrategyFactory().create(muleContext, "workQueue_mb");
+    startIfNeeded(workQueuePs);
+
+    flow = createFlow(muleContext);
+
+    final ReactiveProcessor processor = p -> Flux.from(p).doOnNext(e -> {
+      Blackhole.consumeCPU(100);
+    });
+
+    directSink = directPs.createSink(flow, publisher -> baseFlux(publisher, processor));
+    emitterSink = emitterPs.createSink(flow, publisher -> baseFlux(publisher, processor));
+    workQueueSink = workQueuePs.createSink(flow, publisher -> baseFlux(publisher, processor));
+
+    Flux.<CoreEvent>create(s -> directPipeline = s, ERROR)
+        .transform(directPs.onPipeline(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+    Flux.<CoreEvent>create(s -> emitterPipeline = s, ERROR)
+        .transform(emitterPs.onPipeline(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+    Flux.<CoreEvent>create(s -> workQueuePipeline = s, ERROR)
+        .transform(workQueuePs.onPipeline(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+
+    Flux.<CoreEvent>create(s -> directProcessor = s, ERROR)
+        .transform(directPs.onProcessor(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+    Flux.<CoreEvent>create(s -> emitterProcessor = s, ERROR)
+        .transform(emitterPs.onProcessor(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+    Flux.<CoreEvent>create(s -> workQueueProcessor = s, ERROR)
+        .transform(workQueuePs.onProcessor(publisher -> baseFlux(publisher, processor)))
+        .subscribe();
+
+    directAllSink =
+        directPs.createSink(flow, publisher -> baseFlux(publisher, directPs.onPipeline(directPs.onProcessor(processor))));
+    emitterAllSink =
+        emitterPs.createSink(flow, publisher -> baseFlux(publisher, emitterPs.onPipeline(emitterPs.onProcessor(processor))));
+    workQueueAllSink =
+        workQueuePs.createSink(flow,
+                               publisher -> baseFlux(publisher, workQueuePs.onPipeline(workQueuePs.onProcessor(processor))));
+
+  }
+
+  private Flux<CoreEvent> baseFlux(Publisher<CoreEvent> publisher,
+                                   Function<? super Flux<CoreEvent>, ? extends Publisher<CoreEvent>> transformFunction) {
+    return Flux.from(publisher)
+        .transform(transformFunction)
+        .doOnNext(event -> ((MonoSink<CoreEvent>) (event.getMessage().getPayload().getValue())).success(event))
+        .onErrorContinue((t, event) -> {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove useless curly braces around statement ([squid:S1602](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1602))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7657&issues=AWjJeTCLotIXnmsUTyuW&open=AWjJeTCLotIXnmsUTyuW)

[ISSUE_KEY]: AWjJeTCLotIXnmsUTyuW